src/core_atmosphere/physics/physics_wrf/module_mp_thompson.F

```diff
@@ -64,7 +64,8 @@ MODULE module_mp_thompson
       use mpas_atmphys_functions, only: gammp,wgamma,rslf,rsif
       use mpas_atmphys_utilities
       use mpas_io_units, only : mpas_new_unit, mpas_release_unit
-      use module_mp_radar
+!     use module_mp_radar
```

Comment:
@ldfowler58 Can remove the commented-out 'USE' statement.

Reply:
removed the commented line, as suggested by Wei.

Reply:
@ldfowler58 I didn't realize the line is in module_mp_thompson.F. Will it be able to see mp_radar module in physics_mmm/ directory? @mgduda Can you comment?

Reply:
The compiler should be able to find the `mp_radar` module in `physics_mmm`, since the compilation rules in `src/core_atmosphere/physics/physics_wrf/Makefile` have `-I../physics_mmm`.

Reply:
@mgduda Thanks! I will approve this PR now.

Reply:
The compiler does find mp_radar. I actually tested this change, and found not differences in the reflectivity diagnostics compared to using module_mp_radar.

Reply:
@ldfowler58 Thanks!